.cirrus.yml

```diff
@@ -194,27 +261,20 @@ success_task:
   alias: success
   depends_on:
     - "build"
-    - "build_cross"
+    - "build_aarch64"
     - "validate"
+    - "validate_aarch64"
     - "verify_vendor"
+    - "verify_vendor_aarch64"
     - "unit"
+    - "unit_aarch64"
     - "integration"
+    - "integration_aarch64"
     - "meta"
     - "ubuntu20_build"
     - "centos9_build"
+  ec2_instance: *standard_build_ec2_aarch64
   env:
     CIRRUS_SHELL: "/bin/sh"
   clone_script: *noop
-  bin_cache: *ro_bin_cache
-  # The paths used for uploaded artifacts are relative here and in Cirrus
-  script:
-    # Cross-compiled binary artifact is required by other CI systems
-    - curl --fail --location -O --url https://api.cirrus-ci.com/v1/artifact/build/${CIRRUS_BUILD_ID}/build_cross/binary.zip
-    - unzip binary.zip
-    - rm -f binary.zip
-    - mv bin/* ./
-    - rm -rf bin
-  # Upload tested binary for consumption downstream
-  # https://cirrus-ci.org/guide/writing-tasks/#artifacts-instruction
-  binary_artifacts:
-    path: ./*netavark*
```

Comment:
Please leave this, it ensures we store the build artifacts when Cirrus runs on the branch, but only after all tasks are successful.

I'm not opposed to using your copr repos during setup of various repos (instead of downloading this zip) but I still want to produce the zip in case it's needed for some reason later.

Reply:
done